benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -1012,6 +1012,7 @@
   {"LiveFrom": "10/06/2020", "CaseTypeID": "Benefit", "CaseFieldID": "jointPartyAddressSameAsAppellant", "UserRole": "caseworker-sscs", "CRUD": "CRUD"},
   {"LiveFrom": "10/06/2020", "CaseTypeID": "Benefit", "CaseFieldID": "jointPartyAddress", "UserRole": "caseworker-sscs", "CRUD": "CRUD"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "otherParties", "UserRole": "caseworker-sscs", "CRUD": "CRUD"},
+  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "CaseFieldID": "otherParties", "UserRole": "caseworker-sscs-superuser", "CRUD": "CRUD"},
```

Comment:
You don't need this as it inherits from the "caseworker-sscs" permission on line above